### DISCUSSION TITLE
I can not install it

### DISCUSSION THREAD

**Discussion by willamecm83** [cotes2020/jekyll-theme-chirpy#1202] [Q&A] [answered]
hello,

I would like some help, I do everything in the documentation about installing the theme, but I am not able to see the error.

I copy the repository, clone it, put the bundle command, configure the url, timezone, lag, test it with the command:

undle exec jekyll s

with the local server, it opens the site, but when I put it in github pages it doesn't work, it returns me --- layout: home # Index page ---

if someone help me, thanks.

**Comment by ghost** [comment, marked as answer]
I'm a beginner, and I ran into the same issue when I tried publishing through GitHub Pages.

**Here's how I solved the issue**
1. Make sure you run the command `bundle exec jekyll s` inside the local cloned repository for it to generate the `_site` directory. After that, add your github.io repository in the url line insite `_config.yml` file.
2. You will find this line inside the config file. Change the username to the correct one you have.
```
# fill in the protocol & hostname for your site, e.g., 'https://username.github.io'
url: "https://username.github.io/"
``` 
3. Lastly, execute 'git add', 'git commit', and 'git push' to update your changes. After that, head over to the 'Actions' tab to see if the deployment was successful.

It'll take a few minutes to update. You can also try clearing your browser cache and then revisit your github.io page.
